### ISSUE TITLE
[BUG] Error with number 77

### ISSUE THREAD

**Issue by Citrona** [Cyrexxis/RokTracker#43]
**Describe the bug**
When the OCR reads "77", it writes "717", I've noticed that this problem only occurs in the "Rss Gathered" field
Edit: Sometimes same problem for "Helps" field, and for number finish per "1", script write "11"

**To Reproduce**
Just run a scan lol

**Expected behavior**
The script should write "77" and not "717"

**Your Setup (please complete the following information):**
 - OS: Android
 - Bluestacks Version: 5.11.1.1002
 - Android of Bluestacks: Pie 64-bit
 - Tracker Version: v3.0.0

**Comment by Cyrexxis** [comment]
Hm good to know, but I don't know how much can be done for that. Maybe i can try some more teseract settings or different pre processing. The problem is you can't auto detect if those fileds messed up (i think this can also happen for the "dead" field).

**Comment by Citrona** [comment]
Yes I know it's complicated to fix, I haven't noticed any error on the "dead" fields yet, I checked each field of my scan and the error doesn't happen all the time it's strange
